src/doom/g_game.c

```diff
@@ -2240,6 +2240,10 @@ void G_DoPlayDemo (void)
         fastparm = 0;
         nomonsters = 0;
         consoleplayer = 0;
+
+        // Ability to force -nomonsters and -respawn for playback of v1.2 demos
```

Comment:
Hm, doesn't this need a `gameversion <= exe_doom_1_2` check then? And probably a comment to `stdout`, e.g. "Applying '-respawn' and '-nomonsters' parameters to demo playback".